code/modules/mob/living/carbon/human/human.dm

```diff
@@ -903,4 +908,19 @@
 	if(s_store)
 		add_screams(s_store.alternate_screams)
 	if(wear_id)
-		add_screams(wear_id.alternate_screams)
\ No newline at end of file
+		add_screams(wear_id.alternate_screams)
+
+//I'm not even sure where this function went in our code... It's in RR's PR so ???
```

Comment:
Probably no point in keeping it, couldn't even find out where it was used in RR's PR